packages/example/src/hooks/useKey.ts

```diff
@@ -0,0 +1,14 @@
+import { useEffect } from 'react';
+import RemoteControlManager from '../components/remote-control/RemoteControlManager';
+import { SupportedKeys } from '../components/remote-control/SupportedKeys';
+
+export const useKey = (key: SupportedKeys, callback: (pressedKey: SupportedKeys) => boolean) => {
```

Comment:
Maybe we can add a little tsdoc ?

```suggestion
/**
 * A convenient hook to listen to a key and react to it
 *
 * @example useKey(SupportedKeys.Back, () => { console.log('pressed back!') })
 */
export const useKey = (key: SupportedKeys, callback: (pressedKey: SupportedKeys) => boolean) => {
```